### DISCUSSION TITLE
[bug] it's not for emacs

### DISCUSSION THREAD

**Discussion by tx46** [neovide/neovide#1425] [General]
#### STEPS TO REPRODUCE

1. Start Emacs.
2. Look at screen.

#### EXPECTED RESULTS

Beautiful animations.

#### ACTUAL RESULTS

User starts crying.

*PS. On a more serious note, is there a project like this for Emacs? This is absolutely amazing - great work and big thanks to all the contributors!*

**Comment by MultisampledNight** [comment]
Thank you, I'm sure we all appreciate it! As this is not a real bug, I'll close this issue and convert it to a discussion, if you don't mind.

_PS. I don't know much about Emacs, except than it being full-blown do-anything display-anything. I imagine writing another GUI for it to be quite hard, but I have no idea._

**Comment by A7R7** [comment]
Hi! have a look at https://github.com/manateelazycat/holo-layer !

![neo emacs](https://github.com/neovide/neovide/assets/100180619/b5a67d9c-589d-4baa-af1f-a252ccb2ff6d)
